features/step_definitions/dashboard_steps.rb

```diff
@@ -41,8 +43,18 @@ def top_traffic_for_name (name, opts = {})
 end
 
 When(/^I (fold|unfold) service "([^"]*)"$/) do |action, service_name|
-  step %{service "#{service_name}" is #{action == 'fold' ? 'unfolded' : 'folded'}}
+  step %(service "#{service_name}" is #{action == 'fold' ? 'unfolded' : 'folded'})
 
   service = service_for_name(service_name)
   service.find('.DashboardSection-toggle').click
 end
+
+When(/^the (products|backends) tab is selected$/) do |tab|
```

Comment:
I think that `When... is selected` implies no action of the user involved, so I'd expect it to be assertion instead of a click. I understand the purpose of this step is not to be an assertion, so maybe better `I select on the (product|backends) tab`?